### PR TITLE
CPP-954 Strip whole base directory from S3 upload key

### PR DESCRIPTION
**Pull request by ivomurrell**
# Description

Tool Kit has a plugin to upload assets within a folder to S3. You can set a directory option to specify where in your project the public assets are stored. The plugin is meant to strip out the directory part of the path from the destination key so that you aren’t forced to have a `public/assets/blah/` folder structure in S3 that mirrors your project’s directory structure. However, previously, only the first directory path is stripped from the key, so files would still have a path like `assets/blah/foo.txt`.

I think this is safe to release as a patch instead of a breaking change because the previous behaviour was surprising and I don't think anyone was relying on it.

Closes #284

# Checklist:

- [x] My branch has been rebased onto the latest commit on main (don't merge main into your branch)
- [x] My commit messages are [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/), for example: `feat(circleci): add support for nightly workflows`, `fix: set Heroku app name for staging apps too`

### CODE REVIEW

**Review by juanSanchezAlcala — APPROVED** [review]

File: package-lock.json [hunk 1]
```diff
@@ -20581,7 +20581,7 @@
     },
     "plugins/pa11y": {
       "name": "@dotcom-tool-kit/pa11y",
-      "version": "0.3.2",
```
Comment (joelcarr) [inline package-lock.json:20584]: **question:** Is this change required in this commit?

Comment (ivomurrell) [reply package-lock.json:20584]: this change happens automatically after running `npm install`. release-please isn't able to update the lockfiles when it merges releases so it will have to get updated manually instead. I'd consider it a minor change that we can smuggle into other PRs/commits to avoid us constantly manually updating it in dedicated PRs (and remembering to exclude it from staging in other unrelated commits.)

**Review by joelcarr — APPROVED** [review]

**Comment by ivomurrell** [comment]
[I can't find](https://cs.github.com/?scopeName=All+repos&scope=&q=org%3AFinancial-Times+%40dotcom-tool-kit%2Fupload-assets-to-s3) any apps which were relying on the previous behaviour of only stripping the first folder in the path (every repository so far is just using the default `public` option which will be the folder that is stripped,) so should be safe to release this as a fix.
